### PR TITLE
Reduce the Louisiana exempt income in 2021

### LINKED ISSUE #4453: Reduce the Louisiana exempt income
This only applied in 2021 based on the [tax form](https://revenue.louisiana.gov/TaxForms/IT540i(2021)%20Instructions.pdf#page=9)

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #4453

### CODE REVIEW

**Comment by PavelMakarchuk** [comment]
I can not find the legal code reference for the reduction rate, as the rate was repealed in 2021 will address in #4455

File: policyengine_us/parameters/gov/states/la/tax/income/exempt_income/reduction/rate.yaml
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/la/tax/income/exempt_income/reduction/rate.yaml:1]: rate.yaml confused me as a name - i thought it was a single rate. don't we normally label these marginal structures reduction.yaml? we could then skip the folder and just check if the top rate is nonzero. or leave as-is but rename reduction.yaml within the reduction folder?

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/la/tax/income/exempt_income/reduction/rate.yaml
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/la/tax/income/exempt_income/reduction/rate.yaml:1]: The idea was to have a folder dedicated to everything that is related to the reduction -> one file for the rate and one file for in-effect. I think we have done it similarly in a couple of places

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/la/tax/income/exempt_income/reduction/rate.yaml:1]: usually we name the file just `reduction.yaml` when it is the only file related to the reduction aspect of the given program. If there are multiple pieces we sometimes revert to reduction/rate.yaml etc.

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**